pyproject.toml

```diff
@@ -25,5 +25,5 @@ catkin-pkg = "^1.0.0"
 
 
 [build-system]
-requires = ["poetry-core"]
+requires = ["poetry-core", "setuptools"]
```

Comment:
Why do you update this line?

Reply:
I checked and found that the code was unnecessary, so I deleted it.